apps/pollbook/frontend/src/voter_confirm_screen.tsx

```diff
@@ -239,38 +239,41 @@ export function VoterConfirmScreen({
                     </LabelledText>
                   )}
                 </Row>
-                {hasMailingAddress(voter) && (
-                  <LabelledText
-                    style={{ width: '100%' }}
-                    label={
-                      voter.mailingAddressChange ? (
-                        <s>Mailing Address</s>
-                      ) : (
-                        'Mailing Address'
-                      )
-                    }
-                  >
-                    <VoterMailingAddress
-                      voter={voter}
-                      style={
-                        voter.mailingAddressChange && {
-                          textDecoration: 'line-through',
+                {(hasMailingAddress(voter) || voter.mailingAddressChange) && (
```

Comment:
<img width="1145" height="506" alt="Screenshot 2025-08-11 at 5 38 26 PM" src="https://github.com/user-attachments/assets/2a4b3ab3-595b-4ca7-8b1d-131b78df273e" />

Reply:
did you test this when there isn't an original mailing address?